shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -122,12 +123,9 @@
         new SecureHdfsValidationTask("worker"), WORKER_TASKS);
 
     // ssh validations
-    registerTask("ssh.masters.reachable",
-        "validate SSH port on masters are reachable",
-        new SshValidationTask("masters"), COMMON_TASKS);
-    registerTask("ssh.workers.reachable",
-        "validate SSH port on workers are reachable",
-        new SshValidationTask("workers"), COMMON_TASKS);
+    registerTask("ssh.nodes.reachable",
+        "validate SSH port on all Alluxio nodes are reachable",
+        new SshValidationTask(), COMMON_TASKS);
```

Comment:
do we gain anything by combininb these two?

Reply:
To me the gain is we get rid of some hard-coded "masters" and "workers".